libkineto/src/output_membuf.h

```diff
@@ -97,7 +97,7 @@ class MemoryTraceLogger : public ActivityLogger {
   std::vector<const ITraceActivity*> activities_;
   std::vector<std::unique_ptr<const ITraceActivity>> wrappers_;
   std::vector<std::pair<DeviceInfo, int64_t>> deviceInfoList_;
-  std::vector<std::pair<ResourceInfo, int64_t>> resourceInfoList_;
+  std::vector<std::pair<libkineto::ResourceInfo, int64_t>> resourceInfoList_;
```

Comment:
I don't think you need this namespace. Also how does DeviceInfo not require it but they are both defined in the same namespace?

Reply:
yes, fixed